examples/lib/stories/collision_detection/multiple_shapes.dart

```diff
@@ -234,5 +247,27 @@ class MultipleShapes extends BaseGame
       '${fps(120).toStringAsFixed(2)}fps',
       Vector2(0, size.y - 24),
     );
+    const superDebug = false;
```

Comment:
This could be a dashbook property instead, what do you think?

Reply:
It shouldn't really be exposed to the user I think, it was just so that I could find bugs.
Could be removed completely I guess, but pretty handy to turn it on to help debugging.